<?xml version="1.0"?>

<launch>
<arg name="test_mode" default="False"/>
  
  <node pkg="robot_wheel_speeds" type="robot_wheel_speeds" name="wheel_speeds" respawn="true" output="screen">
    <param name="test_mode" value="$(arg test_mode)" />
    <rosparam file="$(find robot_wheel_speeds)/param/wheel_params.yaml" command="load" />
  </node>
</launch>
